Add new terms for glossary: RHEL software management (#261)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

*See also*:

[discrete]
[[scriptlet-directive]]
==== image:images/yes.png[yes] scriptlet directive (noun)

> *Description*: A _scriptlet_ is a fragment of code in a SPEC file, such as a shell script, that runs during build time ("build scriptlet") or transaction time ("transaction scriptlet") to change the user’s system. Build scriptlets are more limited than transaction scriptlets because build scriptlets do not receive arguments and can only be shell scripts.

Reviewer: ok that there's no mention of a "directive" in desc?

Reviewer 2: +1

Author: Thank you for pointing that out, checked with SMEs -> changing it to lowercase in all occurrences

> *Description*: A _scriptlet_ is a fragment of code in a SPEC file, usually a shell script, which is executed during build time (build scriptlets) or transaction time (transaction scriptlets) in order to make some additional changes to the user’s system. Build scriptlets are more limited than transaction scriptlets because build scriptlets do not receive arguments and can only be shell scripts.

Reviewer 3: ```suggestion
*Description*: A _scriptlet_ is a fragment of code in a SPEC file, such as a shell script, that runs during build time ("build scriptlet") or transaction time ("transaction scriptlet") to change the user’s system. Build scriptlets are more limited than transaction scriptlets because build scriptlets do not receive arguments and can only be shell scripts.
```

*See also*: xref:target[target]

[discrete]
[[source-rpm]]
==== image:images/yes.png[yes] Source RPM (noun)

> *Description*: _Source RPM_, or SRPM, is an RPM package that contains the complete source code, including any patches, and a SPEC file that describes how to build the source code into a binary RPM.

Reviewer 4: Should SPEC be spec (as in other descriptions)?

Author: Thank you! I checked with the sst writer and it should be capitalized: SPEC

*Use it*: yes

*Incorrect forms*: srpm

*See also*: xref:spec-file[spec file], xref:rpm[RPM]

// Fuse: Added "In Red Hat Fuse," and removed "In Fuse tooling,"
[discrete]
[[source-tab]]

> ==== image:images/yes.png[yes] Source tab (noun)

Reviewer 4: I'm guessing that this was an unintentional inclusion. This was removed purposefully by another person in #231.
